robot/packages/cloud_package_manager.go

```diff
@@ -271,13 +276,26 @@ func (m *cloudManager) Cleanup(ctx context.Context) error {
 		expectedPackageDirectories[pkg.thePackage.LocalDataDirectory(m.packagesDir)] = true
 	}
 
-	topLevelFiles, err := os.ReadDir(m.packagesDataDir)
+	allErrors = commonCleanup(m.logger, expectedPackageDirectories, m.packagesDataDir)
+	if allErrors != nil {
+		return allErrors
+	}
+
+	allErrors = multierr.Append(allErrors, m.mlModelSymlinkCleanup())
+	return allErrors
+}
+
+func commonCleanup(logger logging.Logger, expectedPackageDirectories map[string]bool, packagesDataDir string) error {
```

Comment:
also might want to move out of cloud_package_manager into utils... Not sure

Reply:
done in 4e854649